### PR TITLE
Use attributes of CommandContext in Chat.filter

### PR DESCRIPTION
**Pull request by AnnikaCodes**
This does not change the parameters passed to individual filters, in the name of backwards compatibility with old filters.

### CODE REVIEW

File: server/chat.ts [hunk 1]
```diff
@@ -1376,21 +1376,22 @@ export const Chat = new class {
...
 	readonly filters: ChatFilter[] = [];
-	filter(
-		context: CommandContext,
-		message: string,
-		user: User,
-		room: Room | null,
-		connection: Connection,
-		targetUser: User | null = null
-	) {
+	filter(context: CommandContext, message: string) {
```
Comment (Zarel) [inline server/chat.ts:1379]: I think I'd prefer this the other way around: `message, context`.

File: server/chat-commands/info.ts [hunk 2]
```diff
@@ -2616,7 +2616,7 @@ export const commands: ChatCommands = {
 
 		if (this.broadcasting) {
 			if (isPMOrPersonalRoom) {
-				target = Chat.filter(this, target, user, room, connection, this.pmTarget)!;
+				target = Chat.filter(this, target)!;
```
Comment (Zarel) [inline server/chat-commands/info.ts:2619]: I saw a `context.filter` function defined. Can it be used in these places? Just `target = this.filter(target)!;`?

File: server/chat.ts (hunk 1 above)
Comment (AnnikaCodes) [reply server/chat.ts]: Okay.

File: server/chat-commands/info.ts (hunk 2 above)
Comment (AnnikaCodes) [reply server/chat-commands/info.ts]: Yeah, it can.

**Comment by Zarel** [comment]
Thanks! This looks good!

**Pull request merged**